utils/http/utils.go

```diff
@@ -54,3 +55,98 @@ func GetConsoleApiToken(console, port string) (string, error) {
 
 	return token.GetApiToken(), nil
 }
+
+type StreamingConfig struct {
+	Addition    bool
+	Deletion    bool
+	EntityTypes []string //Node, Container, ContainerImage, Pod, KubernetesCluster, CloudNode
```

Comment:
Is it "all" if the list is empty?

Reply:
List should have at-least one entity. 
If it's empty, we treat it as error.

Reply:
Fixed.  We need atleast one entity type.